pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -2123,12 +2159,14 @@ func (cc *XController) manageQueueJob(qj *arbv1.AppWrapper, podPhaseChanges bool
 
 		} else if qj.Status.CanRun && qj.Status.State == arbv1.AppWrapperStateActive {
 			//set appwrapper status to Complete or RunningHoldCompletion
-			derivedAwStatus := cc.getAppWrapperCompletionStatus(qj)
+			derivedAwStatus, genericItemsCompletionStatus := cc.getAppWrapperCompletionStatus(qj)
 
 			//Set Appwrapper state to complete if all items in Appwrapper
 			//are completed
 			if derivedAwStatus == arbv1.AppWrapperStateRunningHoldCompletion {
 				qj.Status.State = derivedAwStatus
+				klog.V(1).Infof("[>>>>>>>>>>] Setting ItemCompletionStatus 1")
```

Comment:
nit: Likely don't need the extra formatting here `>>>>>>`.   Suggest the function name is provided instead.